src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import { getArticleID } from "../utils";
+import { Card, Stack, Badge } from "react-bootstrap";
+import { FaRegComments } from "react-icons/fa";
+import { LuVote } from "react-icons/lu";
+import { useParams } from "react-router-dom";
+
+export default function ArticlePage() {
+  const [currentArticle, setCurrentArticle] = useState({});
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticleID(article_id).then((data) => {
+      const { article } = data;
+      setCurrentArticle(article);
+    });
+  }, []);
+
+  let date = null;
+
+  if (currentArticle.created_at) {
+    const dateCreated = new Date(currentArticle.created_at);
+    date = dateCreated.toLocaleDateString();
+  }
+
+  return (
+    <>
```

Comment:
Could we view the article image within this section? The page could maybe do with a bit more breathing space at the moment, as there's a lot of text going on 🙂